client/src/components/DataFiles/DataFilesModals/DataFilesAddProjectModal.module.scss

```
@@ -0,0 +1,6 @@
+.root {
+  .add-workspace-btn {
+    padding-left: 1.5em;
+    padding-right: 1.5em;
+  }
+}
```

__Overly-Specific CSS__

This nesting is unnecessary. (Reference: [UI - CSS Style Guide - Specificity @ Nesting](https://confluence.tacc.utexas.edu/display/UP/UI+-+CSS+Style+Guide+-+Specificity#UI-CSSStyleGuide-Specificity-Nesting).

```suggestion
.add-workspace-btn {
  padding-left: 1.5em;
  padding-right: 1.5em;
}
```

> P.S. If you are trying to scope `.add-workspace-btn` to prevent conflict, then don't worry about it; because, a CSS module (which this is, because of the `.module.scss` extension) are already scoped to the component which loads it.